doc/setup.md

```diff
@@ -165,3 +165,18 @@ make load_demo_data
 ```
 
 This also adds `testuser@cesium-ml.org` as an administrator.
+
+### nginx configuration
+
+When running on a public server, it can be useful to use X. 509 certificates for Transport Layer Security (TLS) encryption. For example, services such as Let's Encrypt (https://letsencrypt.org/) provide free services to create such certificates when deploying publicly.
```

Comment:
```suggestion
When running a public server, you will likely want to deploy an SSL certificate (i.e., serve `https://your.url` instead of `http://your.url`). Certificates can be obtained for free from services such as Let's Encrypt (https://letsencrypt.org/).
```